wger/weight/views.py

```diff
@@ -42,7 +44,7 @@
 from wger.weight import helpers
 from wger.utils.helpers import check_access
 from wger.utils.generic_views import WgerFormMixin
-
+import fitbit
```

Comment:
pay attention to the order of imports, it is inbuilt, external then app with each category sorted in alphabetical order